mbus-api/src/v0.rs

```diff
@@ -475,3 +547,274 @@ impl From<i32> for ReplicaState {
         }
     }
 }
+
+/// Volume Nexuses
+///
+/// Get all the nexuses with a filter selection
+#[derive(Serialize, Deserialize, Default, Debug, Clone)]
+pub struct GetNexuses {
+    /// Filter request
+    pub filter: Filter,
+}
+
+/// Nexus information
+#[derive(Serialize, Deserialize, Default, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Nexus {
+    /// id of the mayastor instance
+    pub node: String,
+    /// name of the nexus
```

Comment:
The name or the UUID? they  *_MAYBE_*  the same, but don't have to be

Reply:
Yeah should just be the UUID indeed. I think the only one where we use an actual name is the pool, which is the name the user assigns to the pool.

Reply:
Yes its an "oops" that got in and it started to lead its own life. Let us make sure we correct in the new API.